sites/docs/src/content/docs/tutorials/nf-core_components/using_seqera_containers.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Using Seqera Containers"
+subtitle: "Step by step guide to using Seqera Containers in nf-core modules"
+weight: 30
+---
+
+[Seqera Containers](https://seqera.io/containers) simplifies the container creation process for nf-core modules. The following steps will guide you through obtaining both Docker and Singularity containers from the Seqera website to properly configure your module for integration with nf-core workflows.
+
+0. (Optional, but recommended) test building the `environment.yml` locally to ensure that the packages and their versions are compatible:
+
+   ```bash
+   conda env create -f environment.yml
+   ```
+
+1. Go to [https://seqera.io/containers/](https://seqera.io/containers/)
+2. Add all the packages in the search bar (you can copy paste e.g. `conda-forge::<tool>=<version>` from your `environment.yml` file, to make sure you have exactly the right channels/versions).
+3. Make sure container setting is on `Docker`.
+4. Press 'Get Container'.
+5. Copy the displayed URL and put it on the last line of your module's `main.nf` container definition (this does _not_ have a `https://` at the beginning!).
+6. Repeat steps 2-5.
```

Comment:
This is just a suggestion, but I was confused by having to repeat the steps with `Docker` again
```suggestion
6. Repeat steps 2-5 by switching the 'Container settings' to `Singularity`.
```

Reply:
isn't it just to get all the different dependencies?

Reply:
🤔 I don't understand this. You add all the dependencies in point 1, so why do you need to repeat this?

Reply:
I misunderstood this step. @jfy133 what are we repeating here? Because to get the singularity build I can just switch the engine without needing to reenter anything.

Reply:
Agreeing with @mashehu you just need to switch the engine

Reply:
But why would you want to repeat step 2-5 here?